File: IQ/Classes/Ajax/SetFieldOverrideRequest.vb
Imports System.Web.Http
Imports System.Net.Http

Public Class SetFieldOverrideRequest
    Inherits HttpRequestMessage
    Public lid As UInt64
    Public BranchPath As String
    Public ScreenId As Integer
    Public FieldId As Integer
    Public ForceVisibilityTo As Boolean?
    Public ForceOrderTo As Integer?
    Public ForceWidthTo As Double?
    Public ForceSortTo As String
    Public ForceFilterTo As String
End Class

File: IQ/Classes/Ajax/SetFieldOverrideRequest.cs
using System.Web.Http;
using System.Net.Http;

public class SetFieldOverrideRequest : HttpRequestMessage
{
	public UInt64 lid;
	public string BranchPath;
	public int ScreenId;
	public int FieldId;
	public bool? ForceVisibilityTo;
	public int? ForceOrderTo;
	public double? ForceWidthTo;
	public string ForceSortTo;
	public string ForceFilterTo;
}
